src/errors/SimpleInputValidationError.ts

```diff
@@ -0,0 +1,9 @@
+// In the case of non-ajv input validation, use this for input errors.
+export class SimpleInputValidationError extends Error {
```

Comment:
I mentioned this elsewhere but I'm not sure we should have non-ajv input validation.

We don't have to define a full blown type -- we can just use `ajv.validate({ type: 'integer'}, value), or whatever!